x-pack/plugins/security_solution/public/management/components/artifact_card_grid/artifact_card_grid.tsx

```diff
@@ -100,6 +100,29 @@ export const ArtifactCardGrid = memo<ArtifactCardGridProps>(
       [callerDefinedCardProps, onExpandCollapse]
     );
 
+    const isEverythingExpanded = useMemo(() => {
+      for (const [_, currentCardProps] of callerDefinedCardProps) {
+        const currentExpandedState = Boolean(currentCardProps.expanded);
+        if (!currentExpandedState) {
+          return false;
+        }
+      }
+      return true;
```

Comment:
you could just return `Boolean(currentCardProps.expanded)` right?

Reply:
Not sure at all, I'm trying to find some not expanded card. If there is at least one collapsed card, then it returns false, else if everything is expanded, then it returns true.